devito/function.py

```diff
@@ -659,6 +662,7 @@ def __init__(self, *args, **kwargs):
 
             self.time_dim = kwargs.get('time_dim')
             self.time_order = kwargs.get('time_order', 1)
+            self.save = type(kwargs.get('save', None) or None)
```

Comment:
Note for the future: no need the ``, None)`` when you do ``.get`` on a dict. It's the default return value anyway.